python/graphstorm/model/embed.py

```diff
@@ -340,6 +414,75 @@ def require_cache_embed(self):
         """
         return self.cache_embed
 
+    def create_wholememory_optimizer(self, optimizer_type: str, param_dict: dict):
+        """Create a wholegraph sparse optimizer.
+
+        If we use wholegraph to store sparse embeddings, for future update, a joint
+        wholegraph sparse optimizer has to be created ahead of time, and then attach
+        to the (wholegraph)sparse embedding.
+
+        Parameters
+        ----------
+        optimizer_type: str
+            optimizer types: [ "sgd" | "adam" | "adagrad" | "rmsprop" ]
+        param_dict: dict
+            parameters of the optimizer
+
+        Returns
+        -------
+        WholeMemoryOptimizer obj (pylibwholegraph.torch.WholeMemoryOptimizer)
+        """
+        import pylibwholegraph.torch as wgth
+
+        return wgth.create_wholememory_optimizer(optimizer_type, param_dict)
```

Comment:
WholeMemoryOptimizer is cool!

Reply:
Btw, your comment reminds me that there is a bug in WholeMemoryOptimizer that affect the accuracy when embed_dim is not multiple of 32 (https://github.com/rapidsai/wholegraph/pull/108).  I patch fixed it when I perform all the unit tests and e2e tests.